ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java

```diff
@@ -0,0 +1,51 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.config;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.config.ComponentProperties;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.config.LdioConfigurator;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiOutput;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFLanguages;
+import org.apache.jena.riot.RDFWriter;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+
+import java.util.HashMap;
+
+@Configuration
+public class LdioKafkaOutAutoConfig {
+
+	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldio.LdioKafkaOut")
+	public LdioConfigurator ldiKafkaOutConfigurator() {
+		return config -> {
+			final Lang lang = getLang(config);
+			final String topic = config.getProperty("topic");
+			final var kafkaTemplate = createKafkaTemplate(config.getProperty("bootstrap-servers"));
```

Comment:
so far, we don't really use the `var` keyword. So, for consistency reasons, I'd suggest `KafkaTemplate<String,String>`

Reply:
This was a topic on a dev meeting that we were  going to allow it if the variable is clear on itself as it improves readability of the codeblock

Reply:
This was a topic on a dev meeting that we were going to allow it if the variable is clear on itself as it improves readability of the codeblock